checker/src/main/java/org/checkerframework/checker/index/lowerbound/LowerBoundAnnotatedTypeFactory.java

```diff
@@ -162,6 +165,36 @@ private void addLowerBoundTypeFromValueType(
         }
     }
 
+    /** chars are unsigned implies chars are non-negative. See JLS 4.2. */
+    private void ensureCharNonNegative(AnnotatedTypeMirror type) {
+        TypeKind typeKind = type.getUnderlyingType().getKind();
+        switch (typeKind) {
+            case CHAR:
+                if (qualHierarchy.isSubtype(NN, type.getAnnotationInHierarchy(UNKNOWN)))
+                    type.replaceAnnotation(NN);
+                break;
+            case DECLARED:
+                String qualifiedName =
+                        TypesUtils.getQualifiedName((DeclaredType) type.getUnderlyingType())
```

Comment:
When lines end up broken like this I usually try to extract a local variable to make the code more readable. In this case, you could extract the cast: `DeclaredType declared = (DeclaredType) type.getUnderlyingType();`. Then I suspect the line starting with `String qualifiedName = ...` will not be broken.